### DISCUSSION TITLE
Mobile Widget broken on Xiaomi HyperOS

### DISCUSSION THREAD

**Discussion by FelixmaxYT** [community/community#145203] [Mobile]
It opens the app and makes me select my account but then just refreshes the launcher and doesn't add the widget 

Anyone encountered the same issue?

**Comment by bewuethr** [comment]
I'm on Android 13, and the widget just keeps saying "Sign in", though I'm signed in to the app.

**Reply by NASTYA1733** [reply]
Peinlich y you be joy ]()

**Comment by FelixmaxYT** [comment]
Did a little digging and turns out it actually crashes my launcher.

**Comment by gaurav7902** [comment]
In Funtouch OS widget is added but doesn't refresh automatically and even after refreshing manually the contributions don't appear.
Needs to again add the widget to see the latest contributions.

**Comment by iskae** [comment]
Hi @FelixmaxYT 👋 

Thank you so much for reaching out and we are sorry that you are experiencing trouble with the widgets 😞 We are currently investigating the issue and get back to you as soon as possible 🙇 

Thanks!

**Reply by MikaTech-dev** [reply]
Just thought I should say this exact issue still occurs on my Xiaomi Phone that's on HyperOs 1, Android 13.
